src/_data/catalog/index.json

```diff
@@ -65,6 +39,32 @@
     ],
     "id": "material"
   },
+  {
+    "name": "Material 2 components",
+    "description": "Widgets implementing the <a href=\"https://m2.material.io/design\">Material 2 Design guidelines</a>.",
+    "pagecontent": "Material 3 (M3), the latest version of Material Design, is Flutter's default as of Flutter 3.16. Material 2 will eventually be deprecated. Check out the <a href=\"/ui/widgets/material/\">M3 widget catalog</a>.",
+    "subcategories": [
+      {
+        "name": "App structure and navigation"
+      },
+      {
+        "name": "Buttons"
+      },
+      {
+        "name": "Input and selections"
+      },
+      {
+        "name": "Dialogs, alerts, and panels"
+      },
+      {
+        "name": "Information displays"
+      },
+      {
+        "name": "Layout"
+      }
+    ],
+    "id": "material2"
+  },  
```

Comment:
Is this intentional? It doesn't seem related to isolates.

Reply:
no, I don't know how this happened. I will revert